ledger-api/grpc-definitions/com/daml/ledger/api/v1/package_service.proto

```diff
@@ -13,15 +13,29 @@ option java_package = "com.daml.ledger.api.v1";
 option csharp_namespace = "Com.Daml.Ledger.Api.V1";
 
 // Allows clients to query the DAML-LF packages that are supported by the server.
+// ``INVALID_ARGUMENT`` gRPC error will be returned in case of malformed payloads and missing required fields.
+// ``NOT_FOUND`` will be returned in case a request specifies a ledger id that is not valid.
```

Comment:
Is this line required here since it's described for the individual service methods?
We seem to have a mix of sometimes all errors are documented for each method, and sometimes we have common errors documented for services.

Reply:
I'll remove this. If in any place I've left error descriptions at the service definition level, it's by mistake. I intend to have them described at methods definitions.